src/coreclr/src/System.Private.CoreLib/src/System/GC.cs

```diff
@@ -120,6 +120,12 @@ internal enum GC_ALLOC_FLAGS
         [MethodImpl(MethodImplOptions.InternalCall)]
         internal static extern ulong GetGenerationSize(int gen);
 
+        [MethodImpl(MethodImplOptions.InternalCall)]
+        internal static extern ulong GetGenerationTimeBetweenGC(int gen);
+
+        [MethodImpl(MethodImplOptions.InternalCall)]
+        internal static extern int GetGenerationLastGCDuration(int gen);
```

Comment:
If this information is important for telemetry, should we consider putting it on GCMemoryInfo?
cc: @maoni

Reply:
GCMemoryInfo already includes pause info. I don't think we want to include duration because for BGC it's not useful; for non BGC it's basically the same as pause. 

the user of the GetGCMemoryInfo API would detect the GC frequency to the precision their sampling interval allows, eg if they call this API once every second and detected 2 gen0 GCs, they don't know how long exactly elapsed between the 2 GCs but they know they have 2 gen0 GCs in 1s.